app/models/ncr.rb

```diff
@@ -0,0 +1,9 @@
+module Ncr
+  # Make sure all table names use 'ncr_XXX'
+  def self.table_name_prefix
+    'ncr_'
+  end
+
+  EXPENSE_TYPES = %w(BA60 BA61 BA80)
```

Comment:
I was going back-and-forth about moving them with this PR...debatable whether it's NCR-wide information or if it's specific to work orders. Strong feelings?